CONTRIBUTING.md

```diff
@@ -0,0 +1,231 @@
+# Contributing to Prism Themes
+
+Thank you for coming on board!
+
+There are a few ways you can contribute to Prism Themes, such as:
+
+-   [Filing bug reports](#filing-bug-reports)
+-   [Discussing the project](#discussing-the-project)
+-   [Submitting Pull Requests](#submitting-pull-requests), whether it's for documentation or code
+-   [Creating new themes](#creating-new-themes)
+
+## Filing bug reports
+
+If you run into an error or a bug while using a theme from this repository:
+
+1.  Confirm that the bug lies with the theme and not Prism or one of its plugins. You can check by swapping the theme out for another theme. If the issue lies with Prism or one of its plugins, please head over to the [main repository](https://github.com/PrismJS/prism) instead.
+    -   In addition, please be sure that you only have one theme loaded! Having multiple themes loaded may result in conflicting or unexpected styles.
+
+2.  Check [existing issues (both open and closed)](https://github.com/PrismJS/prism-themes/issues?q=is%3Aissue) to see if it had previously been reported and/or resolved. If an existing issue exists, please leave a comment on it instead of creating a new issue.
+
+3.  [Open a new issue](https://github.com/PrismJS/prism-themes/issues/new/choose) with a clear and concise title, and include any relevant information such as:
+    -   The name of the theme you're having an issue with
+    -   Any plugins that you're using, if applicable
+    -   Screenshots and/or a reproducible sample
+    -   Browser information
+
+### Security issues
+
+In the case of security issues, please check our [Security Policy](SECURITY.md) for details instead.
```

Comment:
I think it's better to link the page GitHub gives us.

```suggestion
In the case of security issues, please check our [Security Policy](https://github.com/PrismJS/prism-themes/security/policy) for details instead.
```

Reply:
Oh nice! I didn't know GitHub has a dedicated page for it.